operators/hack/run-e2e.sh

```diff
@@ -33,7 +33,22 @@ sed \
     kubectl apply -f -
 
 # retrieve pod responsible for running the job
-pod=$(kubectl get pods -n $NAMESPACE --selector=job-name=$JOB_NAME --output=jsonpath={.items..metadata.name})
+pod=""
+retry=0
+e2e_pod_creation_timeout=30
+while true; do
+    if [[ ${retry} -ge ${e2e_pod_creation_timeout} ]]; then
```

Comment:
here we are comparing a number of retries with a number of seconds? Seems a bit weird.
I do see we are sleeping 1 seconds but we have no real guarantee that the `kubectl get pods` command is instantaneous.
Should we pick either a total time to wait, or a total number of retries?

Reply:
I will change the variable names to `e2e_pod_creation_attempts`. It has not been designed to be accurate but just to give an other chance to get the pod name.
Most of the time the second attempt is the good one.